src/pylexibank/commands/check_cognates.py

```diff
@@ -0,0 +1,44 @@
+"""
+Check cognates table
+"""
+import functools
+from collections import defaultdict
+from cldfbench.cli_util import with_datasets, add_catalog_spec
+from pylexibank.cli_util import add_dataset_spec, warning
+
+
+def register(parser):
+    add_dataset_spec(parser, multiple=True)
+    add_catalog_spec(parser, 'glottolog')
+
+def check(ds, args):
+    warn = functools.partial(warning, args, dataset=ds)
+    cldf = ds.cldf_reader()
+    
+    if cldf["CognateTable"].common_props.get('dc:extent') == 0:
+        # NOTE: this is using the metadata to identify if we
+        # have cognates or not. Is there a better way?
+        return
+    
+    args.log.info('checking {0} - cognates'.format(ds))
+    
+    # do cognate sets span different words?
+    # i.e. we want 'global' cognate ids so cognate sets should be 
+    # found in one word/parameter only.
+    cognates = {}
+    for row in cldf['CognateTable']:
+        cognates[row['Form_ID']] = row['Cognateset_ID']
+    
+    cogs_by_param = defaultdict(set)
+    for row in cldf['FormTable']:
+        cogset = cognates.get(row['ID'])
+        if cogset:
+            cogs_by_param[cogset].add(row['Parameter_ID'])
+        
+    for c in cogs_by_param:
+        if len(cogs_by_param[c]) > 1:
+            warn('Cognate set {0} is not global: {1}'.format(c, cogs_by_param[c]))
```

Comment:
The cognate set component in CLDF explicitly allows cross-semantic cognate sets. So this check enforces a lexibank-specific assumption. But I think even in lexibank, this assumption may not always hold, e.g. as soon as we include partial cognacy. So maybe we should state the assumption explicitly in some metadata, and incorporate the uniqueness check in `pylexibank` proper?

Reply:
Yeah, I can see that local cognates could be very useful in many datasets, so maybe this check is a bad idea. 

Maybe a better solution is to try detect this during `cmd_makecldf` and add a flag to README.md? e.g. "Cognacy: local -- cognate sets can belong to multiple parameters" or "Cognacy: global -- cognate sets are contained within one parameter". Then we at least know this information, rather than enforcing/validating something we're not sure we want to do. 

(cc @LinguList ?)

Reply:
Ah, good idea. A warning in `makecldf` is probably the best place to address this - and there should be a way for datasets to explicitly ignore the warning.

Reply:
Now I feel like I'm overthinking this -- the cldf/lexibank dataset should faithfully track the data. It is irrelevant if the authors of that data have global or local cognates. Users of the dataset need do their due diligence and figure this out. I'll close this for now.

Reply:
I think it's good to have the framework for these checks setup. Maybe the next step now is rebuilding all datasets and turn any irregularity encountered doing this into a check.